.github/workflows/main.yml

```diff
@@ -36,11 +36,12 @@ jobs:
     steps:
       - uses: actions/checkout@v3
 
-      - id: 'auth'
-        name: 'Authenticate to Google Cloud'
-        uses: 'google-github-actions/auth@v0'
+      - id: "auth"
+        name: "Authenticate to Google Cloud"
+        uses: "google-github-actions/auth@v0"
+        if: env.credentials_json != null
```

Comment:
I'm not 100% clear on where the credentials_json comes from in this step. 
Wouldn't a test for env.GOOGLE_APPLICATION_CREDENTIALS suffice?

Reply:
Ah good call. `with` doesn't use env variables, so my approach won't work. 😢